pkgs/racket-doc/json/json.scrbl

```diff
@@ -157,10 +158,20 @@ the @rfc for more information about JSON.
       "sandwich sandwich" (code:comment "invalid JSON")
       (λ () (read-json))))
 
+  (eval:error
+    (with-input-from-string
+      "false sandwich" (code:comment "valid JSON prefix, invalid remainder is not (immediately) problematic")
+      (λ () (read-json))))
+
```

Comment:
Did you mean to remove `eval:error` here, since there's no error?

Reply:
Oops -- quite right. Fixed.